DOC-761: Suggested edits to the 5.7 changelogs (#6510)
tinymce/tinymce: modules/tinymce/changelog.txt

Version 5.7.0 (TBD)
    Added IPv6 address support to the URI API. Patch contributed by dev7355608 #GH-4409

>     Added new data to the `TableModified` event to describe whether the table `structure`, `style`, or both was modified #TINY-6643

Reviewer: Are you sure about this, reading it aloud sounds really strange?

Author: ```suggestion
    Added two optional return parameters to the `TableModified` event indicating if the `structure` or `style` was modified #TINY-6643
```
?

Reviewer: `Optional` is incorrect as they are always provided. They also aren't `return parameters`, they are properties in the event object/type.

Reviewer: Also, it's not exclusively either-or, it can be any of the following:
- `{ style: false, structure: true }` (structure change)
- `{ style: true, structure: false }` (style change)
- `{ style: true, structure: true }` (both structure and style change)

Reviewer: ```suggestion
    Added new properties to the `TableModified` event to describe whether the table `structure`, `style`, or both were modified #TINY-6643
```

Reviewer 2: ```suggestion
    Added new properties to the `TableModified` event to indicate if the table `structure` or `style` were modified #TINY-6643
```
removed whether so it doesn't imply it's mutually exclusive, so that we maybe don't need the `or both`? wondering if the backticks are needed...

Reviewer 2: ```suggestion
    Added new `structure` and `style` properties to the `TableModified` event to describe how the table was modified #TINY-6643
```
or something maybe? rearranging the sentence...

Reviewer: LGTM :+1:

To clarify, I was originally asking about changing `were` to `was` as that seemed wrong. So this really just spiralled out of control sorry :disappointed:

Reviewer 2: it read weird to me, I only figured out why when I saw your comments

>     Added a new locked column feature to the `table` plugin, blocking column operations on locked columns #TINY-6765

Reviewer: This line was actually meant to be removed. It seems we forgot to?
```suggestion
```

    Added `video` and `audio` live embed support for the `media` plugin #TINY-6229
    Added the ability to resize `video` and `iframe` media elements #TINY-6229

>     Added new `font_css` setting for adding fonts to both the editor and the parent document #TINY-6199

Reviewer: ```suggestion
    Added a new `font_css` setting for adding fonts to both the editor and the parent document #TINY-6199
```

    Fixed an issue where parsing malformed comments could cause an infinite loop #TINY-6864
    Fixed incorrect return types on `editor.selection.moveToBookmark` #TINY-6504
    Fixed the type signature for `editor.selection.setCursorLocation()` incorrectly allowing a node with no `offset` #TINY-6843
    Fixed incorrect behavior when editor is destroyed while loading stylesheets #INT-2282
    Fixed figure elements incorrectly splitting from a valid parent element when editing the image within #TINY-6592
    Fixed inserting multiple rows or columns in a table cloning from the incorrect source row or column #TINY-6906
    Fixed an issue where new lines were not scrolled into view when pressing Shift+Enter or Shift+Return #TINY-6964
    Fixed an issue where list elements would not be removed when outdenting using the Enter or Return key #TINY-5974
    Fixed an issue where file extensions with uppercase characters were treated as invalid #TINY-6940

>     Fixed dialog block messages were unable to be translated #TINY-6971

Reviewer 2: This one reads weird still... `Fixed dialog block message could not be translated`? Not sure that's any better.

Author: @metricjs was it that they were hardcoded so we couldn't get them translated?

Reviewer: The strings used in the `api.block(message)` call were never passed through the TinyMCE translation system. So not sure hard-coded is the right word, but it'd be equivalent.

Reviewer: If not using one of our plugins, you could have technically done something like this to workaround it, which is why I said hard-coded wouldn't be right:
```
api.block(editor.translate(message))
```

Reviewer 2: could we just do this?
```suggestion
    Fixed dialog block messages were not translated #TINY-6971
```
maybe implies the translations were missing instead, but it's not incorrect...

Reviewer: Yeah, that definitely sounds like they are being translated now, which is wrong as we haven't got them translated yet.

Reviewer 2: yeah... 
```suggestion
    Fixed dialog block messages were not passed through TinyMCE's translation system #TINY-6971
```
just be explicit?
